review/comment/comment.go

```diff
@@ -32,9 +35,16 @@ const Ref = "refs/notes/devtools/discuss"
 // FormatVersion defines the latest version of the comment format supported by the tool.
 const FormatVersion = 0
 
+// ErrInvalidRange inidcates an error during parsing of a user-defined file
+// range
+var ErrInvalidRange = errors.New("invalid file location range. The required form is SL[+SC][:EL[+EC]]")
```

Comment:
I think the `SL[+SC][:EL[+EC]]` description needs more explanation.

Maybe add a qualifier at the end of the sentence like ``` where "SL" is the starting line, "SC" is the starting column, "EL" is the ending line, and "EC" is the ending column; with each number starting from 1.```